packages/contracts/test/StabilityPoolTest.js

```diff
@@ -3394,325 +2450,27 @@ contract('StabilityPool', async accounts => {
       assert.isTrue(G_2.gt(G_1))
     })
 
-    it("withdrawETHGainToTrove(), partial withdrawal: doesn't change the front end tag", async () => {
-      await openTrove({ extraLUSDAmount: toBN(dec(10000, 18)), ICR: toBN(dec(10, 18)), extraParams: { from: whale } })
-
-      // A, B, C open troves 
-      await openTrove({ extraLUSDAmount: toBN(dec(10000, 18)), ICR: toBN(dec(2, 18)), extraParams: { from: A } })
-      await openTrove({ extraLUSDAmount: toBN(dec(20000, 18)), ICR: toBN(dec(2, 18)), extraParams: { from: B } })
-      await openTrove({ extraLUSDAmount: toBN(dec(30000, 18)), ICR: toBN(dec(2, 18)), extraParams: { from: C } })
-      
-      // A, B, C, D, E provide to SP
-      await stabilityPool.provideToSP(dec(10000, 18), frontEnd_1, { from: A })
-      await stabilityPool.provideToSP(dec(20000, 18), frontEnd_2, { from: B })
-      await stabilityPool.provideToSP(dec(30000, 18), ZERO_ADDRESS, { from: C })
-
-      // Defaulter opens a trove, price drops, defaulter gets liquidated
-      await openTrove({  ICR: toBN(dec(2, 18)), extraParams: { from: defaulter_1 } })
-      await priceFeed.setPrice(dec(105, 18))
-      assert.isFalse(await th.checkRecoveryMode(contracts))
-      await troveManager.liquidate(defaulter_1)
-      assert.isFalse(await sortedTroves.contains(defaulter_1))
-
-      await th.fastForwardTime(timeValues.SECONDS_IN_ONE_HOUR, web3.currentProvider)
-
-      // Check A, B, C have non-zero ETH gain
-      assert.isTrue((await stabilityPool.getDepositorETHGain(A)).gt(ZERO))
-      assert.isTrue((await stabilityPool.getDepositorETHGain(B)).gt(ZERO))
-      assert.isTrue((await stabilityPool.getDepositorETHGain(C)).gt(ZERO))
-
-      await priceFeed.setPrice(dec(200, 18))
-
-      // A, B, C withdraw to trove
-      await stabilityPool.withdrawETHGainToTrove(A, A, { from: A })
-      await stabilityPool.withdrawETHGainToTrove(B, B, { from: B })
-      await stabilityPool.withdrawETHGainToTrove(C, C, { from: C })
-
-      const frontEndTag_A = (await stabilityPool.deposits(A))[1]
-      const frontEndTag_B = (await stabilityPool.deposits(B))[1]
-      const frontEndTag_C = (await stabilityPool.deposits(C))[1]
-
-      // Check deposits are still tagged with their original front end
-      assert.equal(frontEndTag_A, frontEnd_1)
-      assert.equal(frontEndTag_B, frontEnd_2)
-      assert.equal(frontEndTag_C, ZERO_ADDRESS)
-    })
-
-    it("withdrawETHGainToTrove(), eligible deposit: depositor receives LQTY rewards", async () => {
```

Comment:
keep this test?

Reply:
this is removed as the LQTY events have been removed from stability pool state changes